src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/WorkPackageSummary.tsx

```diff
@@ -94,8 +91,7 @@ const WorkPackageSummary: React.FC<WorkPackageSummaryProps> = ({ workPackage })
               )}
             </Grid>
             <Grid item xs={6}>
-              <Typography fontWeight="bold">Deliverables:</Typography>
-              <Typography>{deliverablesList}</Typography>
+              <DetailDisplay label="Deliverables" content={deliverablesList} paddingRight={1} />
```

Comment:
Yeah I would say a list like this is an instance where the DetailDisplay isn't applicable